lib/diagnostic.js

```diff
@@ -5,22 +5,36 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+  let result = arguments[0];
+  for (let i=1;i<arguments.length;i++){
```

Comment:
Make sure you start your 'for loop' from 0, not 1. There will be times that you may want to start your for loops from a number other than 0, but you almost always want to start there and build towards array.length (especially in these first few weeks)! You'll end up tripping up people working with your code--mostly `future you`!